src/applications/disability-benefits/526EZ/Form526EZApp.jsx

```diff
@@ -5,15 +5,18 @@ import formConfig from './config/form';
 
 import ITFWrapper from './containers/ITFWrapper';
 import DisabilityGate from './containers/DisabilityGate';
+import EVSSClaimsGate from './containers/EVSSClaimsGate';
 
 export default function Form526Entry({ location, children }) {
   return (
-    <ITFWrapper location={location}>
-      <DisabilityGate>
-        <RoutedSavableApp formConfig={formConfig} currentLocation={location}>
-          {children}
-        </RoutedSavableApp>
-      </DisabilityGate>
-    </ITFWrapper>
+    <EVSSClaimsGate location={location}>
+      <ITFWrapper location={location}>
+        <DisabilityGate>
+          <RoutedSavableApp formConfig={formConfig} currentLocation={location}>
+            {children}
+          </RoutedSavableApp>
+        </DisabilityGate>
+      </ITFWrapper>
+    </EVSSClaimsGate>
```

Comment:
I totally agree. The alternative is handling all these discreet tasks in one big wrapper, but I figured it would make for easier reasoning this way. Also, if we have to extract or modify something later, it should be pretty straightforward.

But yeah. I totally agree this is getting silly.